reana_server/config.py

```diff
@@ -161,22 +162,23 @@ def _(x):
 # ===========================
 
 
-def _is_valid_rate_limit(rate_limit: str) -> bool:
-    return bool(
-        re.match(r"[0-9]+(\sper\s|\/)(second|minute|hour|day|month|year)", rate_limit)
-    )
-
-
 def _get_rate_limit(env_variable: str, default: str) -> str:
     env_value = os.getenv(env_variable, "")
-    return env_value if _is_valid_rate_limit(env_value) else default
+    try:
+        parse(env_value)
+        return env_value
+    except ValueError:
+        return default
 
 
 # Note: users that are connecting via reana-client will be treated as guests by the Invenio framework
 RATELIMIT_GUEST_USER = _get_rate_limit("REANA_RATELIMIT_GUEST_USER", "20 per second")
 RATELIMIT_AUTHENTICATED_USER = _get_rate_limit(
     "REANA_RATELIMIT_AUTHENTICATED_USER", "20 per second"
 )
+RATELIMIT_PER_ENDPOINT = {
+    "launch.launch": _get_rate_limit("REANA_RATELIMIT_LAUNCH_WORKFLOW", "1/30 second"),
```

Comment:
*thought(non-blocking):* Because we have `launch` Flask blueprint, and the endpoint is also named `launch` the naming is confusing here.